tutorials/pages/send-shimmer-tokens-with-iota-sdk.md

```diff
@@ -0,0 +1,148 @@
+# Send Shimmer tokens with iota-sdk in NodeJs
+
+## Introduction
+
+This tutorial shows how to get Shimmer testnet tokens and send them to another address using javascript/typescript.
+
+### What you will learn
+
+- Create a Stronghold account
+- Generate a Shimmer address
+- Get some Shimmer testnet tokens
+- Fetch your balance
+- Send tokens to another address
```

Comment:
Would it make sense to use some more neutral wording here? So we don't need to change the tutorial again once it is available on IOTA?

Reply:
Should it be like `L1` instead of `Shimmer`?

Can also do a separate PR to update Shimmer -> L1 once it's available on IOTA.

Reply:
I think calling it L1 could make sense 👍

Reply:
What about this change? Do you want to fix it now or later? I would do it now. Otherwise you need to touch it again in 2 weeks :trollface:

Reply:
I was thinking if we should even do it or not.
The tutorial can remain for Shimmer I guess. It'll be specific in that sense. IOTA can follow the same cause it'll be have same updates as Shimmer.

But if we call it L1 in general, it'd also imply for other L1s like Ethereum, Solana, etc. Should we do it?

(P.S. don't think it's gonna be just 2 weeks 😅)

Reply:
Why would a IOTA wiki provide tutorials for other L1s? 🤔

Reply:
But yeah we can leave it as it is. No strong opinion from my side